Content/StructureResolver.php

```diff
@@ -133,14 +156,19 @@ public function resolveProperties(
                     );
                 }
             } else {
-                if (!$structure->hasProperty($sourceProperty)) {
+                $propertyStructure = $structure;
+                if ('title' === $sourceProperty) {
+                    $propertyStructure = $originalStructure;
+                }
+
+                if (!$propertyStructure->hasProperty($sourceProperty)) {
                     continue;
                 }
 
-                $property = $structure->getProperty($sourceProperty);
+                $property = $propertyStructure->getProperty($sourceProperty);
 
                 $contentView = $this->resolveProperty(
-                    $structure,
+                    $propertyStructure,
```

Comment:
think we can access the structure of the property via the `getStructure()` method